src/lib/TranscriptEditor/MediaPlayer/PlaybackRate.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './PlaybackRate.module.css';
+class PlaybackRate extends React.Component {
+    constructor(props) {
```

Comment:
I think we don't need this unless you're setting some extra state. But no harm.

Reply:
Ok, removing all the unnecessary constructors